apps/src/componentLibrary/typography/typography.module.scss

```diff
@@ -1,7 +1,7 @@
 @import "color", "font";
 
 /* This file is a adopted copy of shared/css/typography.scss
-  updating styles in this file will require a manual sync with the that file
+  updating styles in this file will require a manual sync with the that file.
```

Comment:
super nits:
- `an adopted`
- Separate sentences, so end the first with a `.`. and start the next with `Updating`